triggers/k8s/airflow-operator.yaml

```diff
@@ -21,12 +21,11 @@ github:
   pullRequest:
     branch: .*
     commentControl: COMMENTS_ENABLED
-id: e84b1610-5061-49af-b8e0-62665d2369c5
```

Comment:
Please consider to keep id. AFAIK [import](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/triggers#update-an-existing-trigger) command works with name, however id seems to be more safe to rely on.

Reply:
The ids are generated anyway. The names are better to make sure we have no repeated triggers.

Reply:
If you take a look on this command implementation, you will see that first id is comparing, later names. It means that id is more solid. For example, can be changed in the UI. Please note that this repo should host up-to-date triggers and should as a backup in case it's needed.